Makefile

```diff
@@ -15,7 +15,7 @@ build: generate-prod
 
 .PHONY: lint
 lint: generate-prod
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.2
```

Comment:
Think this can be bumped to `1.54.2`
https://github.com/golangci/golangci-lint/releases